packages/venia-concept/src/ServiceWorker/registerRoutes.js

```diff
@@ -61,10 +67,19 @@ export default function() {
      * Route for all JS files and bundles. This route uses CacheFirst
      * strategy because if the file contents change, the file name will
      * change. There is no point in using StaleWhileRevalidate for JS files.
+     *
+     * After 1 week, the files will be purged from cache.
      */
     workbox.routing.registerRoute(
         new RegExp(/\.js$/),
-        new workbox.strategies.CacheFirst()
+        new workbox.strategies.CacheFirst({
+            cacheName: RUNTIME_CACHE_NAME,
+            plugins: [
+                new workbox.expiration.Plugin({
+                    maxAgeSeconds: ONE_WEEK
```

Comment:
Would definitely want to confirm this with @zetlen. It seems reasonable that if you haven't visited a site in the past week, offline mode is not going to work (you'd have pretty stale data anyway), but worth double checking. What was the reason for the new TTL?

Reply:
Just to delete old files, save on cache storage memory. Ideally changing this should not affect the app. We can set it to INFINITY and the app should still work as expected.